test/models/work_test.rb

```diff
@@ -0,0 +1,225 @@
+      end
+
+      it "returns work updated more recently if same title (possible with diff. cat) and same number of votes" do
+        sleep (1)
+        @newer_work.save
+        expect(Work.spotlight).must_equal @newer_work
+        expect(Work.spotlight.votes.count).must_equal 0
+        expect(Work.spotlight.votes.count).must_equal works(:workBA).votes.count
+        expect(Work.spotlight.title).must_equal works(:workBA).title
+      end
+    end
+
+    describe "sort_cat" do
+      it 'returns empty array for when there are no works for a certain category' do
+        Work.delete_by(category: 'book')
+        expect(Work.sort_cat('book')).must_be_empty
+        expect(Work.sort_cat('book')).must_be_kind_of Array
+      end
+      it 'returns an array of works in a category sorted by vote, most to least ' do
+        # two book entries
+        #@work1.save
+        #@work2.save
+
+        # vote for one
+        Vote.create(work_id: works(:workBB).id, user_id: @user.id)
+
+        # get sorted array
+        sorted_books = Work.sort_cat('book')
+
+        expect(sorted_books.length).must_equal Work.where(category: 'book').count
+        expect(sorted_books.first).must_equal works(:workBB)
+        expect(sorted_books.first.id).must_equal works(:workBB).id
+        expect(sorted_books.first.votes.count).must_equal 1
+        expect(sorted_books.last.votes.count).must_equal 0
+      end
+
+      it "sorts works with the same number of votes in ABC order" do
+        Vote.delete_all # need to clear votes so all have same number
+        # two book entries
+
+        # get sorted array
+        sorted_books = Work.sort_cat('book')
+
+        expect(sorted_books.length).must_equal Work.where(category: 'book').count
+        expect(sorted_books.first).must_equal works(:workBA)
+        expect(sorted_books.first.id).must_equal works(:workBA).id
+        expect(sorted_books.first.votes.count).must_equal 0
+        expect(sorted_books.last.votes.count).must_equal 0
+      end
+    end
+
+    # sort_cat is already tested, will just check for # num of elements and instance types
+    describe "work_hash" do
+      it "returns an hash with empty arrays of values if there are no works" do
+        Work.delete_all
+        empty_arrays = Work.work_hash
+        expect(empty_arrays.length).must_equal 3
+        empty_arrays.each {|cat, array| expect(array).must_be_empty}
+      end
+      it "returns a hash with keys matching to what category the array contains works of" do
+        work_hash = Work.work_hash
+        expect(work_hash.length).must_equal 3
+
+        total_works = 0
+        work_hash.each do |cat, array|
+          expect(array.length).must_equal Work.where(category: cat.to_s.chop).count
+          total_works += array.length
+          array.each {|work| expect(work.category).must_equal cat.to_s.chop}
+        end
+        expect(total_works).must_equal Work.all.count
```

Comment:
Cooool test!!!!! I would probably say that the `total_works` bit is a little redundant with the looped `expect(array.length)...` assert. I'd feel comfortable if you kept the loop and dropped the `total_works` bit!